wremnants/muon_calibration.py

```diff
@@ -225,8 +226,16 @@ def make_jpsi_crctn_unc_helper(filepath, n_scale_params = 3, n_tot_params = 4, n
     )
     hist_scale_params_unc = hist.Hist(axis_eta, axis_scale_params, axis_scale_params_unc)
     for i in range(n_eta_bins):
-        lb, ub = i * n_scale_params, (i + 1) * n_scale_params
-        hist_scale_params_unc.view()[i,...] = var_mat[lb:ub][:]
+        if args.dummyVar:
+            nvar = n_scale_params * n_eta_bins
```

Comment:
np.full. There is some code duplication here with the same thing below, can it be shared?

Reply:
got rid of the duplication by unifying different jpsi crctn unc helpers.